src/main/scala/akka/persistence/cassandra/CassandraSession.scala

```
@@ -67,7 +68,10 @@ private[cassandra] final class CassandraSession(
         val s = initialize(settings.sessionProvider.connect())
         if (_underlyingSession.compareAndSet(null, s)) {
           s.foreach { ses =>
-            CassandraMetricsRegistry(system).addMetrics(metricsCategory, ses.getCluster.getMetrics.getRegistry)
+            Try {
+              if (!ses.getCluster.isClosed())
+                CassandraMetricsRegistry(system).addMetrics(metricsCategory, ses.getCluster.getMetrics.getRegistry)
+            }
```

This will silence any exception here, since nobody looks at that `Try`?